article/Future-work/future-work.tex

```diff
@@ -1 +1,14 @@
-\section{Future work}
\ No newline at end of file
+\section{Future work}
+We have several ideas for future work.
+There are obviously many alternative methods to calculate the weighted summation layer combinations other than ALC and BLC that could be worth investigated.
+Additionally these layer combinations could be based on information about the dataset, so that it is not necessary to run each layer individually.
```

Comment:
```suggestion
Additionally, the distribution of these weights could be based on statistics of the dataset so that it was not necessary to run each layer individually before being able to calculate the weights.
But this would require a more exhaustive analysis of tendencies and patterns in the different datasets which could then be linked to which layers perform best.
```